client/src/helpers/APICalls/getMessagesByConvoId.ts

```diff
@@ -0,0 +1,18 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { Messages } from '../../interface/Message';
+
+interface Props {
+  convoID: string | undefined;
```

Comment:
Why are we using undefined here? :)

Reply:
I will fix this. undefined shouldn't be here.

Reply:
removed as requested.